City-of-Helsinki/berth-reservations-admin
# VEN-473 | Berth details on the Offer page

## Description :sparkles:
![image](https://user-images.githubusercontent.com/23040926/76742288-8c411280-6779-11ea-905a-95328daf7d6b.png)

- Implementation for berth details on the offer page
- Quick filters based on piers on the table's header on the offer page

## Issues :bug:

## Testing :alembic:

- From the sidebar, go to the applications page `Hakemukset`
- Expand one of the rows 
- Click on one of the options under `VALITUT SATAMAT`, it goes to the offer page
- Expand one of the rows to see the berth details component
- Click on one of the filters on the table header to filter the table's values based accordingly, or click on `Kaikki` to clear the filters


**Automated tests ⚙️**

**Manual testing 👷**

## Screenshots :camera_flash:

## Additional notes :spiral_notepad:
